### PR TITLE
Update suspension and unlock email text

### PR DESCRIPTION
**Pull request by chrislo**
Trello: https://trello.com/c/0oTag1bi

This PR updates the content of the suspension reminder, suspension notification and unlock instruction emails following [some user research we did on their content](https://docs.google.com/document/d/1kIedI1QlYwf6RNjbVR3l4HApeRFmoHooShZNrDy7NNk/edit?pli=1).

### CODE REVIEW

File: app/mailers/user_mailer.rb
Comment (floehopper) [inline app/mailers/user_mailer.rb:118]: I know it wasn't you who added it in the first place, but it would be better to use `User.unlock_in` here instead of `1.hour` - I believe the former is provided from the Devise config which I think what this `1.hour` refers to here.

File: test/models/user_mailer_test.rb
Comment (floehopper) [inline test/models/user_mailer_test.rb:159]: Subjective: I tend to use `Time.zone.parse("2023-10-31 02:00:00")` in tests, because I think it's more readable.

**Review by floehopper — APPROVED** [review]
I've added a couple of minor comments, but otherwise this looks good to me! 👍 

I think it would be good to make the `User.unlock_in` change before merging unless I've misunderstood that bit of code, but I'm happy to leave that to your discretion.

As an aside, I note that we have [Devise's `unlock_strategry` set to `:both`](https://github.com/alphagov/signon/blob/243f9eb898eb70be40cab6ffea55009b16f8e7cf/config/initializers/devise.rb#L184-L189) and my reading of the docs is that the idea of sending them the email is to give them the opportunity to unlock their own account (presumably legitimized by the fact they must have access to their email account to see the email). However, we don't seem to include a link to allow the user to unlock their own account. I think it would be interesting to understand whether not including the link was intentional and/or whether it's even sensible. Perhaps adding such a link in the email would be a simple way to reduce load on the Content 2nd Line Support team...?

File: app/mailers/user_mailer.rb
Comment (chrislo) [reply app/mailers/user_mailer.rb]: That's great, thank you for spotting it.

File: test/models/user_mailer_test.rb
Comment (chrislo) [reply test/models/user_mailer_test.rb]: Done, that is nicer, thanks!

**Comment by chrislo** [comment]
I've captured the suggestion to allow users to unlock themselves in [this trello card](https://trello.com/c/HovPBBeK).

**Pull request merged**
